### PR TITLE
View events page

### PR DESCRIPTION
**Pull request by ishak52**

### CODE REVIEW

File: public/css/viewEventsStyle.css [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+.bigContainer {
+display: flex;
+/* height: 200px; */
+flex-direction: column;
+flex-wrap: wrap;
+}
+
+
+
+/* .input {
```
Comment (NoureldeanSaed) [inline public/css/viewEventsStyle.css]: Remove commented code.

File: public/css/viewEventsStyle.css [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+/* margin-right: 90px; */
+/* border-radius: 50%; */
+align-items: center;
+justify-content: space-around;
+width: 260px;
+height: 260px;
+border: 1px solid #828282;
+}
+
+.a {
```
Comment (NoureldeanSaed) [inline public/css/viewEventsStyle.css]: Describe your classnames better.

File: src/controllers/deleteEvents.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+const Events = require('../database/event_schema');
+const dbConnect = require('../database/db_connect');
+
+exports.get = (req, res, next) => {
+
+const eventsId = req.params.id;
+
+Events.findByIdAndRemove({ _id: eventsId })
```
Comment (NoureldeanSaed) [inline src/controllers/deleteEvents.js]: Separate queries into a different directory, a controller has nothing to do with the database -directly-, but through the query, create a function that takes an `id` and removes from the database, and reuse it whenever you feel it.

File: src/controllers/viewEvents.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+const Events = require('../database/event_schema');
+const dbConnect = require('../database/db_connect');
+
+exports.get = (req, res, next) => {
+  Events.find().then((result) => {
```
Comment (NoureldeanSaed) [inline src/controllers/viewEvents.js]: Separate your queries.

File: src/controllers/deleteEvents.js (hunk 3 above)
Comment (ishak52) [reply src/controllers/deleteEvents.js]: @NoureldeanSaed I have already solved this problem in the delete_events branch

File: public/css/viewEventsStyle.css (hunk 1 above)
Comment (ishak52) [reply public/css/viewEventsStyle.css]: @NoureldeanSaed  done :+1:

File: public/css/viewEventsStyle.css (hunk 2 above)
Comment (ishak52) [reply public/css/viewEventsStyle.css]: @NoureldeanSaed  actually i will push the pages style in a separated branch, so i'll change the classnames :+1:

File: src/controllers/index.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
+const router = require('express').Router();
+const viewEvents = require('./viewEvents');
+const home = require('./home');
+const { notFound } = require('./error');
+const deleteEvents = require('./deleteEvents');
+
+// router.get('/viewEvents', viewEvents.get);
+router.get('/viewEvents', viewEvents.get);
+router.get('/deleteEvents/:id', deleteEvents.get);
+// router.delete('/deleteEvents/:id', deleteEvents.delete);
```
Comment (NoureldeanSaed) [inline src/controllers/index.js]: @ishak52 Remove commented code.

File: src/controllers/viewEvents.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
+const dbConnect = require('../database/db_connect');
+const viewQuery = require('../database/queries/view_events_querie');
+
+exports.get = (req, res, next) => {
+  viewQuery((err, result) => {
+    // validate
```
Comment (NoureldeanSaed) [inline src/controllers/viewEvents.js]: Unnecessary comment.

**Pull request merged**